CONTRIBUTING.md

```diff
@@ -99,9 +99,9 @@ npm run fix # Runs both of the above commands in parallel
 
 npm run test:watch # Runs karma on Chrome, re-running when source files change
 
-npm test # Lints all files, runs karma, runs closure tests, and then runs coverage enforcement checks.
+npm test # Lints all files, runs karma, runs typescript tests, and then runs coverage enforcement checks.
 npm run test:unit # Only runs the karma tests
-npm run test:closure # Runs closure build tests against all closurized files
+npm run test:typescript # Runs typescript build tests against all typescript files
```

Comment:
I should've realized this before, but this task doesn't exist, so remove this line